src/transformers/models/bart/modeling_bart.py

```diff
@@ -1369,7 +1369,8 @@ def forward(
         past_key_values_length = past_key_values[0][0].shape[2] if past_key_values is not None else 0
 
         if inputs_embeds is None:
-            inputs_embeds = self.embed_tokens(input) * self.embed_scale
+            inputs_embeds = self.embed_tokens(input)
+        inputs_embeds = inputs_embeds * self.embed_scale
```

Comment:
I don't think we change this -- it is a breaking change; the same `inputs_embeds` input will yield a different output after this PR on a well-established model.

Does it solve any problem from the VLM side?

Reply:
No, afaik these models are not used as backbones in VLMs, so prob no real benefit for now. 

Close the PR?

Reply:
@zucchini-nlp We can keep the PR, it's useful! Not a priority, but since it's done let's add it :D

Let's just revert this line 🤗

Reply:
Hmm that will make fail tests for equivalence between `input_ids` and `input_embeds`. Then the better decision if to disable totally inputs embeds for these models. Okay, will do